sqlalchemy_redshift/dialect.py

```diff
@@ -96,6 +98,36 @@ class RedshiftImpl(postgresql.PostgresqlImpl):
   \s* \) \s*                # Arbitrary whitespace and literal ')'
 """, re.VERBOSE)
 
+# Reserved words as extracted from Redshift docs.
+# Command used to extract:
+# curl -q "http://docs.aws.amazon.com/redshift/latest/dg/r_pg_keywords.html" | tr '\n' '\r' | sed 's/.*\(AES128.*WITHOUT\).*/\1/' | tr '\r' '\n' | sed 's/^\([A-Z0-9_]*\).*/"\1",/' | paste -s -d' ' - | fold -s -w 70 | awk '{print "    "$0}' | tr 'A-Z' 'a-z'  # noqa
```

Comment:
The postgres base dialect has the list hardcoded, so I like keeping with what that's doing. This seems like the simplest approach to make it obvious what's going on, rather than adding indirection by separating it to a different file.